app/main.py

```diff
@@ -1,3 +1,75 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, (int, float)):
+            return Distance(km = self.km + other)
+        elif isinstance(other, Distance):
+            return Distance(km = self.km + other.km)
+
+    def __iadd__(self, other):
+        if isinstance(other, (int, float)):
+            self.km += other
+        elif isinstance(other, Distance):
+            self.km += other.km
+        return self
+
+    def __mul__(self, other):
+        if isinstance(other, (int, float)):
+            return Distance(km = self.km * other)
+
+    def __truediv__(self, other):
+        if isinstance(other, (int, float)):
+            return Distance(km=round(self.km / other, 2))
```

Comment:
The `__truediv__` method should handle division by zero to prevent runtime errors. Consider adding a check to raise a `ZeroDivisionError` if `other` is zero.